### ISSUE TITLE
[Bug]: Hard to translate split sentences in flatpak-cli-transaction.c

### ISSUE THREAD

**Issue by haaninjo** [flatpak/flatpak#4956]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a bug that matches the one I want to file, without success.
- [X] If this is an issue with a particular app, I have tried filing it in the appropriate issue tracker for the app (e.g. under https://github.com/flathub/) and determined that it is an issue with Flatpak itself.
- [X] This issue is not a report of a security vulnerability (see [here](https://github.com/flatpak/flatpak/blob/main/SECURITY.md) if you need to report a security issue).

### Flatpak version

1.13.3

### What Linux distribution are you using?

Debian

### Linux distribution version

Testing

### What architecture are you using?

x86_64

### How to reproduce

Similar to https://github.com/flatpak/flatpak/issues/4877, but starting a new bug rather than overloading that with other affected files.

`app/flatpak-cli-transaction.c` contains two strings that are hard to translate due to being split in several gettext calls.

Arguments against this can be seen at https://wiki.gnome.org/TranslationProject/DevGuidelines/Never%20split%20sentences

Strings in question:
```
Info: %s%s %s is end-of-life, in favor of %s\n
Info: %s%s %s is end-of-life, with reason:\n
```

The first %s is the word `pinned`, which has a single string for translation but will take different endings in Swedish depending on whether the second %s is the translation of `app` or `runtime`.

### Expected Behavior

For Swedish this could be fixed by having `app` or `runtime` explicit in the strings:
```
Info: %sapp %s is end-of-life, in favor of %s\n
Info: %sruntime %s is end-of-life, in favor of %s\n
Info: %sapp %s is end-of-life, with reason:\n
Info: %sruntime %s is end-of-life, with reason:\n
```

### Actual Behavior

Only one translation of pinned is possible, so at present Swedish grammar will be wrong for one of the cases `app` or `runtime`.

**Comment by mwleeds** [comment]
Thanks for the bug report, this was my fault as I was not aware of this faux pas. PR incoming

**Comment by mwleeds** [comment]
As a side note, only runtimes can be pinned not apps, so the particular grammar issue you mentioned could be fixed with that knowledge, but I will un-split the sentences instead.

**Comment by mwleeds** [comment]
Fix available in #4963
